custom_model_runner/datarobot_drum/drum/drum.py

```diff
@@ -344,17 +344,23 @@ def run_test_predict(self):
         self.run_mode = RunMode.SCORE
         self.options.code_dir = self.options.output
         self.options.output = os.devnull
+        __target_temp = None
         if self.options.target:
             __tempfile = NamedTemporaryFile()
             df = pd.read_csv(self.options.input)
+            if self.target_type == TargetType.TRANSFORM:
+                target_df = df[self.options.target]
+                __target_temp = NamedTemporaryFile()
+                target_df.to_csv(__target_temp.name, index=False)
             df = df.drop(self.options.target, axis=1)
             # convert to R-friendly missing fields
             if self._get_fit_run_language() == RunLanguage.R:
                 df = handle_missing_colnames(df)
             df.to_csv(__tempfile.name, index=False)
             self.options.input = __tempfile.name
-        CMRunTests(self.options, self.run_mode, self.target_type).check_prediction_side_effects()
-        self._run_fit_and_predictions_pipelines_in_mlpiper()
+        CMRunTests(self.options, self.run_mode, self.target_type).check_prediction_side_effects(
```

Comment:
so I don't think we necessarily need the "run_fit_and_prediction_pipelines_in_mlpiper" given the prediction side effects test? also, I'm not sure if there's a way to make the server check faster?

Reply:
we don't need to run predict given the side effects test